examples/Tabs.overflow.tsx

```diff
@@ -3,57 +3,78 @@
+import { Tabs } from '@itwin/itwinui-react';
 
 export default () => {
-  const [index, setIndex] = React.useState(0);
-  const getContent = () => {
-    switch (index) {
-      case 0:
-        return 'Bentley Systems, Incorporated, is an American-based software development company that develops, manufactures, licenses, sells and supports computer software and services for the design, construction, and operation of infrastructure.';
-      case 1:
-        return "The company's software serves the building, plant, civil, and geospatial markets in the areas of architecture, engineering, construction (AEC) and operations.";
-      case 2:
-        return 'Their software products are used to design, engineer, build, and operate large constructed assets such as roadways, railways, bridges, buildings, industrial plants, power plants, and utility networks.';
-      case 3:
-        return 'The company re-invests 20% of their revenues in research and development.';
-      case 4:
-        return 'Bentley Systems is headquartered in Exton, Pennsylvania, United States, but has development, sales and other departments in over 50 countries.';
-      case 5:
-        return 'The company had revenues of $700 million in 2018.';
-      case 6:
-        return 'Keith A. Bentley and Barry J. Bentley founded Bentley Systems in 1984.';
-      default:
-        return 'They introduced the commercial version of PseudoStation in 1985, which allowed users of Intergraphs VAX systems to use low-cost graphics terminals to view and modify the designs on their Intergraph IGDS (Interactive Graphics Design System) installations.';
-    }
-  };
-  const labels = [
-    <Tab key={1} label='Bentley Overview' />,
-    <Tab key={2} label='Markets' />,
-    <Tab key={3} label='Uses' />,
-    <Tab key={4} label='Reinvestment' />,
-    <Tab key={5} label='Location' />,
-    <Tab key={6} label='Revenue' />,
-    <Tab key={7} label='Founders' />,
-    <Tab key={8} label='History' />,
-  ];
+  const [active, setActive] = React.useState('Bentley Overview');
+
+  const [tabData] = React.useState([
+    { name: 'Bentley Overview' },
+    { name: 'Markets' },
+    { name: 'Uses' },
+    { name: 'Reinvestment' },
+    { name: 'Location' },
+    { name: 'Revenue' },
+    { name: 'Founders' },
+    { name: 'History' },
+  ]);
 
   return (
-    <div
-      style={{
-        maxInlineSize: 425,
-        border: '1px solid lightpink',
-        padding: 8,
-      }}
-    >
-      <Tabs
-        labels={labels}
-        overflowOptions={{ useOverflow: true }}
-        onTabSelected={setIndex}
-        activeIndex={index}
-      >
-        {getContent()}
-      </Tabs>
+    <div style={{ maxWidth: 425, border: '1px solid lightpink', padding: 8 }}>
+      <Tabs.Wrapper overflowOptions={{ useOverflow: true }}>
+        <Tabs.TabList>
+          {tabData?.map((item, index) => {
+            return (
+              <Tabs.Tab
+                value={`tab${index}`}
```

Comment:
can we use name instead of index as the value?

Reply:
Updated examples

Reply:
why not just `item.name` instead of `tab${item.name}2`? see https://github.com/iTwin/iTwinUI/pull/1548#discussion_r1342689004 as well.